chia/full_node/weight_proof.py

```diff
@@ -1504,6 +1504,10 @@ def __get_rc_sub_slot(
 
     assert segment.rc_slot_end_info is not None
     if idx != 0:
+        # this is not the first slot, ses details should not be included
+        ses_hash = None
+        new_ssi = None
+        new_diff = None
```

Comment:
My understanding is that this was a (somewhat) unrelated bug fix, which I really think we ought to have a test covering as well. Do we?
I don't think a test need to hold up this PR though

Reply:
Yeah it's a bug fix. @almogdepaz knows more about it, but it happens when there is an empty slot at the start of an epoch